doc: working_with_nrf: squash structure for nRF91 (#15139)
nrfconnect/sdk-nrf: doc/nrf/device_guides/nrf91/nrf91_features.rst

Customers who are developing their own hardware with the nRF9160 are strongly recommended to use the `nRF9160 Antenna and RF Interface Guidelines`_ as a reference.
See `GPS interface and antenna`_ for more details on GNSS interface and antenna.

> Thingy:91 has a GNSS receiver, which, if activated, allows the device to be located globally using GNSS signals.

Reviewer: ```suggestion
Thingy:91 has a GNSS receiver, which allows the device to be located globally using GNSS signals if it is activated.
```

> In :ref:`asset_tracker_v2`, GNSS is activated by default.

Reviewer: ```suggestion
In :ref:`asset_tracker_v2`, the GNSS receiver is activated by default.
```

  The application displays tracking and GNSS fix information in the serial console.
* The :ref:`gnss_sample` sample does not use assistance by default but can be configured to use nRF Cloud A-GNSS, P-GPS, or a combination of both.
  The sample displays tracking and fix information as well as NMEA strings in the serial console.

Operating modes
***************

> nRF91 Series devices contain multiple LED patterns, which indicate the operating state of the device as described in the :ref:`LED indication <led_indication>` section of the :ref:`asset_tracker_v2_ui_module`.

Reviewer: ```suggestion
nRF91 Series devices can display multiple LED patterns that indicate the operating state of the device as described in the :ref:`LED indication <led_indication>` section of the :ref:`asset_tracker_v2_ui_module` of the Asset Tracker v2 documentation.
```
